src/components/SavedProjectPlan/ProjectPlanCard.jsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Divider,
+  Flex,
+  Heading,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import StandardCard from "./StandardCard";
+
+const ProjectPlanCard = () => {
+  const [hasProjectPlanCard, setHasProjectPlanCard] = useState(true);
+
+  const defaultProps = {
```

Comment:
good job!! Just extract this into a new file and import it. Call it `StandardCardDefaultProps.js`